azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -273,6 +273,123 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"upgrade_description": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"force_restart": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"health_check_retry_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_check_stable_duration": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_check_wait_duration": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_domain_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_replica_set_check_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"max_percent_unhealthy_applications": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+									"max_percent_unhealthy_nodes": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+								},
+							},
+						},
+						"delta_health_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"max_percent_delta_unhealthy_applications": {
+										Type:     schema.TypeInt,
+										Optional: true,
+									},
+									"max_percent_delta_unhealthy_nodes": {
+										Type:     schema.TypeInt,
+										Optional: true,
+									},
+									"max_percent_upgrade_domain_delta_unhealthy_nodes": {
+										Type:     schema.TypeInt,
+										Optional: true,
+									},
+									"application_delta_health_policy": {
+										Type:     schema.TypeList,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"application_type": {
```

Comment:
and remove application here
```suggestion
												"type": {
```